A/ADEPT/build_tarballs.jl

```diff
@@ -0,0 +1,93 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
+
+name = "ADEPT"
+version = v"1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/mgawan/ADEPT.git",
+        "166e38b234fd4a6c1e756348eee4253b4ed281ce",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/ADEPT
+install_license LICENSE
+mkdir build
+cd build
+export CUDA_HOME=${WORKSPACE}/destdir/cuda
+export PATH=${PATH}:${CUDA_HOME}/bin
+export CUDACXX=${CUDA_HOME}/bin/nvcc
+mv ${WORKSPACE}/destdir/cuda/lib ${WORKSPACE}/destdir/cuda/lib64
```

Comment:
```suggestion
ln -s ${WORKSPACE}/destdir/cuda/lib ${WORKSPACE}/destdir/cuda/lib64
```
Doesn't make sense to do a copy, just link it.